TF_Settings_Web/src/Components/Pages/Camera/CameraPage.tsx

```diff
@@ -0,0 +1,32 @@
+import '../../../Styles/Camera/CameraPage.css';
+
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+
+import CameraCalibratePage from './CameraCalibratePage';
+import CameraPosition, { PositionType } from './CameraPosition';
+import CameraSetupSelection from './CameraSetupSelection';
+import { ManualSetup } from './ManualSetup';
+
+const CameraPage = () => {
+    const [activePosition, setActivePosition] = React.useState<PositionType>(null);
+
+    return (
+        <Routes>
+            <Route path="" element={<CameraSetupSelection />} />
+            <Route
+                path="quick"
+                element={
+                    <CameraPosition
+                        activePosition={activePosition}
+                        setPosition={(position: PositionType) => setActivePosition(position)}
+                    />
+                }
+            />
+            <Route path="quick/calibrate/*" element={<CameraCalibratePage activePosition={activePosition} />} />
```

Comment:
What's the purpose of the `/*` at the end of the path?

Reply:
That's to let the router know that the 'quick/calibrate' route has children (e.g. quick/calibrate/top)